shoop/themes/classic_gray/static_src/js/custom.js

```diff
@@ -29,6 +29,29 @@ function getProductListViewMode() {
     return "grid";
 }
 
+function updateProductPrice(productId) {
+    const $quantity = $("#product-quantity");
```

Comment:
If `custom.js` is not transpiled with Babel (I don't know about the current gulpfile), we can't use `const` safely cross-browser.